.gitpod.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM gitpod/workspace-full@sha256:bb84c4490170fbfdeb577535a6dec5aafd1eb8fdf433f9a344bd8f4a69af5707
+ARG FNSA_VERSION=2.0.1
+
+RUN rustup toolchain install 1.69.0
+RUN rustup default 1.69.0
+RUN rustup target add wasm32-unknown-unknown
+RUN rustup default 1.74.1
+RUN cargo install cosmwasm-check
+RUN cargo install cargo-generate
+RUN rustup default 1.69.0
```

Comment:
Do we need to set the version of cargo-generate and other tools? Because cargo-generate 0.19.0 requires the latest version of rustc. If it updates in the future, like to 0.20.0, an error may occur.